index.html

```diff
@@ -5663,7 +5663,21 @@ <h3>Language and Base Direction</h3>
   "@id": "https://www.w3.org/2018/credentials/examples#title"
 }
         </pre>
+      </section>
+
+      <section class="informative">
+        <h3>Providing Default Language and Direction</h3>
 
+        <p>
+A default language and direction MAY be provided for the entire credential.
+However, doing so introduces a higher demand on creating a compliant processor
+and fully compliant context files. Consequently, using default language and
```

Comment:
Not sure this is about "compliance", which is around syntax... you can have a data model that's fully compliant, but where the language information and semantics are wrong. It's the latter that we're concerned about.

```suggestion
However, doing so introduces a higher demand on context creators and 
downstream processors. Consequently, using default language and
```